arm-recoveryservicesbackup/2016-06-01/swagger/registeredIdentities.json

```diff
@@ -0,0 +1,110 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/registeredIdentities/{identityName}": {
+      "delete": {
+        "tags": [
+          "ProtectionContainers"
+        ],
+        "description": "Unregisters the given container from your Recovery Services Vault.",
+        "operationId": "ProtectionContainers_Unregister",
```

Comment:
1. Registration is a manual step - one has to download the agent manually on the DPM machine and run the registration step. Unregistration can be used when the backup needs to be discontinued but the machine is not accessible. 
2. This is in a separate JSON since in the service, this API is hosted in a separate service. Since we use Swashbuckle to generate the swagger spec, we need to create a JSON file per each service.